diesel_tests/tests/update.rs

```diff
@@ -228,10 +227,30 @@ fn update_with_no_changes() {
         name: None,
         hair_color: None,
     };
-    update(users::table)
+    let update_result = update(users::table).set(&changes).execute(connection);
+    assert!(update_result.is_err());
```

Comment:
Changed this from a panic check, to an `is_err` check, since this now returns `QueryBuilder(EmptyChangeset)`